backend/utils/logger.py

```diff
@@ -62,40 +57,47 @@ class Logger(logging.Logger):
     """Logging class"""
 
     def __init__(self):
-        timestamp = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
-        file_name = f"iceberg-{timestamp}.log"
-        data_path = get_data_path()
-
-        super().__init__(file_name)
-        formatter = logging.Formatter(
+        self.timestamp = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+        self.filename = f"iceberg-{self.timestamp}.log"
+        super().__init__(self.filename)
+        self.formatter = logging.Formatter(
             "[%(asctime)s | %(levelname)s] <%(module)s.%(funcName)s> - %(message)s",
             datefmt="%Y-%m-%d %H:%M:%S",
         )
+        self.logs_dir_path = data_dir_path / "logs"
+        os.makedirs(self.logs_dir_path, exist_ok=True)
 
-        if not os.path.exists(data_path):
-            os.mkdir(data_path)
+        self.addFilter(RedactSensitiveInfo())
+
+        console_handler = logging.StreamHandler()
+        console_handler.setLevel(logging.INFO)
+        console_handler.setFormatter(self.formatter)
+        self.addHandler(console_handler)
+        self.console_handler = console_handler
+        self.file_handler = None
 
-        if not os.path.exists(os.path.join(data_path, "logs")):
-            os.mkdir(os.path.join(data_path, "logs"))
+    def configure_logger(self, debug=False, log=False):

```

Comment:
This is done to avoid circular dependency with importing settings_manager which imports the logger.  Instead we configure the logger from the app entrypoint.